From 755933b2b020fcd264e5493c59bebd8d44b0a49e Mon Sep 17 00:00:00 2001
From: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Date: Tue, 17 Jul 2012 14:35:04 +0200
Subject: [PORT FROM MAIN] Revert "WIFI: Move calibration file to /factory"

BZ: 47543

This reverts commit 5c3d231cc4b97582f2560bbc7ee6b3d57c48546a.
JB integration /factory partition not yet available
To be reverted when /factory available

Change-Id: If3168ba796b6670ce722ebc70782ba0ba117b6ff
Orig-Change-Id: Ieea10fce1a023a4d6de70a252fb5fe2b0a461b85
Signed-off-by: Jonathan DE CESCO <jonathan.de.cesco@intel.com>
Reviewed-on: http://android.intel.com:8080/59018
Reviewed-by: Regairaz, Julie <julie.regairaz@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/77969
Reviewed-by: Li, XiaojingX <xiaojingx.li@intel.com>
Tested-by: Li, XiaojingX <xiaojingx.li@intel.com>
---
 wlan_prov/Android.mk          |    2 +-
 wlan_prov/wlan_provisioning.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wlan_prov/Android.mk b/wlan_prov/Android.mk
index 3b1e09a..ed94446 100644
--- a/wlan_prov/Android.mk
+++ b/wlan_prov/Android.mk
@@ -36,7 +36,7 @@ $(SYMLINKS): $(LOCAL_INSTALLED_MODULE) $(LOCAL_PATH)/Android.mk
 	@echo "Symlink: $@"
 	@mkdir -p $(dir $@)
 	@rm -rf $@
-	$(hide) ln -sf $(addprefix /factory/wifi/,$(NVS_FILE)) $@
+	$(hide) ln -sf $(addprefix /data/misc/firmware/ti-connectivity/,$(NVS_FILE)) $@
 
 ALL_DEFAULT_INSTALLED_MODULES += $(SYMLINKS)
 ALL_MODULES.$(LOCAL_MODULE).INSTALLED := \
diff --git a/wlan_prov/wlan_provisioning.c b/wlan_prov/wlan_provisioning.c
index 1e11073..71422d3 100644
--- a/wlan_prov/wlan_provisioning.c
+++ b/wlan_prov/wlan_provisioning.c
@@ -37,7 +37,7 @@
 const unsigned char NullMacAddr[MAC_ADDRESS_LEN] = { 0, 0, 0, 0, 0, 0 };
 
 #define WIFI_PATH "/data/misc/wifi"
-const char NVS_file_name[] = "/factory/wifi/wl1271-nvs.bin";
+const char NVS_file_name[] = "/data/misc/firmware/ti-connectivity/wl1271-nvs.bin";
 const char Default_NVS_file_name[] = "/system/etc/wifi/wl1271-nvs.bin";
 const char WLAN_SDIO_BUS_PATH[] = "/sys/bus/sdio/drivers/wl1271_sdio/";
 #define WLAN_DRV_SDIO_NAME "wl1271_sdio"
-- 
1.7.4.1

